iosHackthonApp/iosHackthonApp.xcodeproj/project.pbxproj

```diff
@@ -546,7 +566,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
Bumped this because using LazyVstack is only available in ios14 and with a List view I couldn't get rid of the separators